### PR TITLE
Demarcate include blocks (#54)

### PR DESCRIPTION
**Pull request by JiaeK**
Previously include blocks are not visually demarcated.

tyxml v3 page

And after this change, 

- Include blocks are demarcated 
- The border on the right-hand side shows which sections come from included signatures
include open

(also the bottom)
include open bottom

- When hovering, the shade of include blocks changes
include hover

### CODE REVIEW

File: asset/doc.css [hunk 1]
```diff
@@ -85,3 +85,24 @@ div.odoc div.spec tbody td.def {
...
   min-width: 40%
 }
+
+.odoc-include details {
+  position: relative;
+}
+
+.odoc-include details:after {
+  display: block;
+  content: " ";
```
Comment (tmattio) [inline asset/doc.css:95]: Why do we need a `content`?

File: asset/doc.css [hunk 2]
```diff
@@ -5,7 +5,7 @@ div.odoc {
   position: relative;
 }
 
-div.odoc div.spec {
+div.odoc div.spec .spec.include {
```
Comment (tmattio) [inline asset/doc.css]: AFAICT, `div.spec` should work on `.spec.include`, unless the elements that are not `spec include` are not `div`? Why was this change necessary?

**Review by tmattio — COMMENTED** [review]
Thanks a lot @JiaeK, this looks great! I tested locally and it's working. Leaving a few comments for things I'm not sure about and we can merge after.

File: asset/doc.css (hunk 1 above)
Comment (JiaeK) [reply asset/doc.css:95]: Yeah I wasn't sure about this - the previous Tyxml page code has this snippet and it's for the pseudo-element (i.e. `:after`) after all so I thought it would need this part. I'll erase it.

File: asset/doc.css (hunk 2 above)
Comment (JiaeK) [reply asset/doc.css]: It's mainly due to the fact that this issue inquired demarcating the include blocks, so I assumed this part doesn't work for the include blocks particularly, so it needed another CSS selector to detect. I'm sorry, I'll also redo this.

File: asset/doc.css (hunk 1 above)
Comment (tmattio) [reply asset/doc.css:95]: Maybe we do need it, it was just a genuine question 🙂  Let's try removing it and see how the docs looks!

File: asset/doc.css (hunk 2 above)
Comment (tmattio) [reply asset/doc.css]: That's a welcome change, I was just surprised we needed this, because the `.spec.include` should already be a subset of `div.spec`, but if that's not the case, let's keep this change.

Comment (JiaeK) [reply asset/doc.css]: When I simulated it, it looked like it needed and I tried something like with `div.spec`, it didn't detect well.. Shall we also remove this and see how it goes?

**Comment by JiaeK** [comment]
I did the update. Can you test locally again please? @tmattio

**Comment by tmattio** [comment]
The changes were indeed necessary, so fixed the CSS after the last commit, merging now 🙂
